test/integration/consensus/consensus_sunny_day.cpp

```diff
@@ -38,11 +39,16 @@ class FixedCryptoProvider : public MockYacCryptoProvider {
 
   VoteMessage getVote(YacHash hash) override {
     auto vote = MockYacCryptoProvider::getVote(hash);
-    vote.signature = clone(TestSignatureBuilder().publicKey(*pubkey).build());
+    auto signature = std::make_shared<MockSignature>();
+    data = std::make_unique<shared_model::crypto::Signed>("");
```

Comment:
can be local + static
Otherwise on multiple calls of `getVote` there's might be a dangling pointer, if I understand correctly